rpcs3/Emu/Cell/lv2/sys_process.cpp

```diff
@@ -415,12 +415,34 @@ void _sys_process_exit2(ppu_thread& ppu, s32 status, vm::ptr<sys_exit2_param> ar
 		, hdd1 = std::move(hdd1), klic = g_fxo->get<loaded_npdrm_keys>().last_key(), old_config = Emu.GetUsedConfig()]() mutable
 	{
 		sys_process.success("Process finished -> %s", argv[0]);
+
+		auto func = [old_size = g_fxo->get<lv2_memory_container>().size, vec = g_fxo->get<id_manager::id_map<lv2_memory_container>>().vec](u32 sdk_suggested_mem) mutable
+		{
+			// Save LV2 memory containers
+			g_fxo->init<id_manager::id_map<lv2_memory_container>>()->vec = std::move(vec);
+
+			// Empty the containers, accumulate their total size
+			u32 of_containers = 0;
```

Comment:
i don't understand the name of this variable

Reply:
Fixed.